stun_server/region/dns.tf

```
@@ -7,6 +7,7 @@ resource "cloudflare_record" "instance_dns" {
   name       = join("-", ["stun", data.aws_region.current.name])
   content    = data.aws_network_interface.stun_server_interface.association[0].public_ip
   type       = "A"
-  proxied    = true
+  ttl        = 1
```

```suggestion
  ttl        = 600
```

<https://developers.cloudflare.com/dns/manage-dns-records/reference/ttl/#unproxied-records>

---

Changed to 300 as we discussed.